sql/clients_last_seen_v1.sql

```diff
@@ -19,9 +19,26 @@ WITH current_sample AS (
 SELECT
   @submission_date AS submission_date,
   CURRENT_DATETIME() AS generated_time,
+
+  -- This last_date struct is used to record the last date that a particular
+  -- client met various criteria; we record a null date if the client does
+  -- not meet a given criterion.
+  STRUCT (
+    IF(
+      country IN ('US', 'FR', 'DE', 'UK', 'CA'),
```

Comment:
pretty sure this needs to be more specific to work, like so:
```suggestion
      current_sample.country IN ('US', 'FR', 'DE', 'UK', 'CA'),
```